proposals/0001.md

```diff
@@ -0,0 +1,274 @@
+* Title: *Error Handling in Rust*
+* Champion: *@FintanH, fintan.halpenny@gmail.com*
+* References:
+  * [Yoshua Wuyats error handling survey][survey]
+  * [failure][failure]
+  * [snafu][snafu]
+  * [anyhow][anyhow]
+  * [thiserror][thiserror]
+
+**Note**: The links to the packages are to versions that are at the latest as of 31/1/2019.
```

Comment:
Is the date still accurate?

Reply:
It was at the time of writing :smile: I can do another pass and update if needs be.